v2/internal/runner/runner.go

```diff
@@ -105,6 +106,28 @@ func New(options *types.Options) (*Runner, error) {
 			reportingOptions.DiskExporter = &disk.Options{Directory: options.DiskExportDirectory}
 		}
 	}
+	if options.ESExport != "" {
+		if reportingOptions != nil {
+			reportingOptions.ElasticsearchExporter = &es.Options{
+				ESIP:              options.ESExport,
+				ESPort:            options.ESPort,
+				ESSSL:             options.ESSSL,
+				ESSSLVerificaiton: options.ESSSLVerificaiton,
+				ESUsername:        options.ESUsername,
+				ESPassword:        options.ESPassword,
+			}
+		} else {
+			reportingOptions = &reporting.Options{}
+			reportingOptions.ElasticsearchExporter = &es.Options{
```

Comment:
it seems that you could extract the assignment out of the condition without duplicating it

Reply:
@forgedhallpass ok, I just pushed the changes.